substra/sdk/schemas.py

```diff
@@ -155,6 +155,23 @@ def exclusive_paths(cls, values):  # noqa: N805
             raise ValueError("'path' or 'paths' field must be set.")
         return values
 
+    @pydantic.model_validator(mode="before")
+    def resolve_paths(cls, values):  # noqa: N805
+        """Resolve given path is relative."""
+        if "paths" in values:
+            paths = []
+            for path in values["paths"]:
+                path = pathlib.Path(path)
+                paths.append(path.resolve()) if not path.is_absolute() else paths.append(path)
```

Comment:
```suggestion
                paths.append(path.resolve())
```
Do we need to check `is_absolute()`  as`path.resolve()` will return itself if already a absolute path?